main.html

```diff
@@ -1,77 +1,255 @@
+  First, the <strong>Product Owner</strong> determines what functionality the host team is willing to accept as contributions.
+  The <strong>Contributor</strong> is the individual submitting the code contribution.
+  The <strong>Trusted Committer</strong> provides any timely support and mentorship that the contributor needs to successfully submit the code contribution.
+</p>
+
+<p>
+  With those definitions, here is the basic outline of an InnerSource contribution.
+</p>
+<p>
+  <ol>
+    <li>
+      Guest team requests a feature from the host team.
+    </li>
+    <li>
+      Product owner agrees that this feature is desirable in the host team's system.
+    </li> 
+    <li>
+      As the product owner prioritizes work for the host team, items other than the guest team's request are chosen.
+    </li>
+    <li>
+      Host team communicates the (de)prioritization of the guest team's request and asks if they would like to complete it via InnerSource contribution.
+    </li>
+    <li>
+      Guest team agrees to submit the feature request via InnerSource.
+    </li>
+    <li>
+      Product owner ensures that user stories representing the feature request are created.
+      These stories should describe the requested feature in terms agreeable to the guest team.
+      They also list any details from the host team on how the feature should be delivered in order for the work to be accepted.
+      Examples of such details include architecture constraints, coding conventions, dependency usages, data contracts, etc.
+    </li>
+    <li>
+      Trusted committers' time is made available to the guest team when the contributor is working.
+    </li>
+    <li>
+      Supported by the trusted committer(s), the contributor submits the pull request(s) to implement the requested feature.
+    </li>
+    <li>
+      Pull request is merged, and the new feature is completed.
+    </li>
+  </ol>
+</p>
+<p>
+  Note that these steps do not assume a specific system for the general organization of a team's time or priorities.
+  InnerSource assumes that teams already have existing such methods of organization.
+  InnerSource simply provides a framework of how to use them to work together where there is a guest team desiring to contribute code to a host.
+</p>
+
+<h2>
+  Benefits
+</h2>
+
+<p>
+  There are many benefits to collaborating via InnerSource.
+  First and foremost, <strong>the guest team gets their feature</strong> request when they need it without the long-term burden of maintaining it.
+  When the guest team would have spent time on the feature regardless (whether through <em>Workaround</em> or <em>Escalate</em>), InnerSource provides a way to use that time towards developing the feature in a place that's centrally-maintained and shared.
+</p>
+
+<p>
+  While that result is the shining benefit of InnerSource, another result that's easy to overlook is the fact that <strong>the host team gets their feature, too</strong>.
```

Comment:
Why is this easy to overlook?  This whole Oriole should be about a **_guest team_** getting a feature completed with **_a host team_**.  If it isn't clear by this point I think that the message should be more focused on this point.  

Getting internal features complete with as little friction as possible where there are cross team dependencies should be the main goal of Innersource!